conf/messages.en

```diff
@@ -212,17 +216,19 @@ traderDetails.holderOfTransit.contactName.heading = What is the contact’s name
 traderDetails.holderOfTransit.contactName.checkYourAnswersLabel = Contact’s name
 traderDetails.holderOfTransit.contactName.error.required = Enter the contact’s name
 traderDetails.holderOfTransit.contactName.error.length = The contact’s name must be 70 characters or less
-traderDetails.holderOfTransit.contactName.error.invalid = The contact’s name must only include letters a to z without accents, numbers 0 to 9, ampersands (&), apostrophes, asterisks(*), forward slashes, full stops, hyphens, percent signs, question marks and spaces
-traderDetails.holderOfTransit.contactName.change.hidden = contact’s name
-
-traderDetails.holderOfTransit.contactTelephoneNumber.title = What is {0}’s telephone number?
-traderDetails.holderOfTransit.contactTelephoneNumber.heading = What is {0}’s telephone number?
-traderDetails.holderOfTransit.contactTelephoneNumber.checkYourAnswersLabel = Contact’s telephone number
-traderDetails.holderOfTransit.contactTelephoneNumber.hint = For international numbers include the country code
-traderDetails.holderOfTransit.contactTelephoneNumber.error.required = Enter {0}’s telephone number?
-traderDetails.holderOfTransit.contactTelephoneNumber.error.length = {0}’s telephone number must be {1} characters or less
-traderDetails.holderOfTransit.contactTelephoneNumber.error.invalidFormat = {0}’s telephone number must be a valid phone number
-traderDetails.holderOfTransit.contactTelephoneNumber.change.hidden = contact’s telephone number
+traderDetails.holderOfTransit.contactName.error.invalid = The contact’s name must only include letters a to z without accents, numbers 0 to 9, ampersands (&), apostrophes, at signs (@), forward slashes, full stops, hyphens, question marks and spaces
+traderDetails.holderOfTransit.contactName.change.hidden = Contact’s name
+
+traderDetails.holderOfTransit.contactTelephoneNumber.title = What is {0}’s phone number?
```

Comment:
Accordiing to the Content on this ticket this should be just
What is the transit holder’s contact phone number?
i.e. no Dynamic name.